src/seedu/addressbook/data/person/Person.java

```diff
@@ -34,6 +34,20 @@ public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables){
+        String concatenatedStringResult = "";
+
+        for(Printable p: printables) {
+
+            concatenatedStringResult.concat(p.getPrintableString() + " ");
```

Comment:
Another way you can do this is using the `StringJoiner` class that java provides.